### DISCUSSION TITLE
옥은수-부동산-서울 부동산에서의 전세사기 문제와 데이터 분석

### DISCUSSION THREAD

**Discussion by OKEunsu** [dinnercoding/wanted-pre-onboarding-challenge-data#108] [2023년 11월 프리온보딩 데이터 챌린지]
# 도메인을 선택한 이유
- 부동산 도메인을 선택한 이유는 전세 원룸을 구하면서 부동산 시장에서 발생하는 문제에 대한 관심이 높아졌습니다. 특히, 전세사기와 같은 위험이 부각되어 있어 이에 대한 데이터 분석을 통해 사람들에게 유용한 정보를 제공하고자 합니다.
# 구체적인 문제 정의
- 부동산에서 특히, 전세사기 부분에 대한 위험이 강조되고 있습니다. 이러한 문제에 대한 해결책을 찾고, 사람들을 위험에서 보호하기 위해 어떠한 데이터 분석을 진행할지에 대한 목표를 세웠습니다.
# 데이터 업로드
- 서울열린광장 : https://data.seoul.go.kr/dataList/OA-21276/S/1/datasetView.do#
- 서울열린광장 : 실거래가, 전월세가 데이터셋
- 한국부동산원 부동산테크 '임대차 시장 사이렌' 자료
- 강대식 국민의힘 의원실, 서울시 발표자료
- 서울특별시 임차권등기명령 신청 부동산 현황 자료 : 집토스, 법원 등기정보광장 분석
# 데이터 시각화
### 1.1 서울특별시 구별 다세대, 연립 전세가율
전세가율 : 매매가 대비 전세가 비율 -> 전세가율 80%이상인 집 피하기
![newplot (2)](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/7af44af4-6b57-4152-a94f-3fbc7cf16422)

### 1.2 서울특별시 구별 전세사기건수, 금액
![서울특별시 전세사고건수 지도](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/282913fd-1a84-4421-84cf-f70f20e12b6d)
서울특별시 전세사기 건수는 주로 한강 기준으로 서쪽으로 많이 분포를 했다. 그 중 강서구가 주변에 비해 압도적으로 많은 사고가 일어났다. 
![newplot](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/cd04ec3f-4fc2-4126-b04c-3090482f6521)
![newplot (1)](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/1980c4b3-16f5-48cc-a11c-bd7aab9c3348)
상위 5개인 구는 강서구, 구로구, 금천구, 양천구, 관악구, 은평구로 나오고, 하위 5개는 용산구, 종로구, 서초구, 성동구 중구, 동대문구이다.

서울특별시 전세사고건수 : 2,709건
서울특별시 전세사고 금액 : 724,325,350(천원) 

## 2. 서울특별시 전세사고건수, 구별 전세가율 산점도
![전세가율, 사고건수 산점도](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/a3e7666a-a1f6-4a2a-9982-6c638dcbc0c3)
서울특별시 구별 사고건수, 전세가율 상관계수 값은 0.407이다. 전세가율와 전세사고건수는 상관관계가 있다.
## 3. 연령대 별 피해자 결정 신청, 접수 현황
20대 에서 60대로 산출
![연령대별 전세사기 접수](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/015423ff-e310-4a11-9c27-334572e75c22)
![연령대별 전세사기 접수 분포](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/8c9baaaf-b319-4af1-8228-9090c37edf49)
20,30대 사회초년생이 주로 전세사기에 대한 접수를 진행을 했으며 이는 76.2%으로 절반이상이다.

## 4. 서울특별시 임차권등기명령 신청 현황
![다운로드](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/d9ec88ca-8daf-4fa3-ac7f-bc9ee8894453)
임차권등기명령이란 보증금을 돌려 받지 못한 임차인을 보호하기 위한 방법,  
대항력, 우선 변제권이 유지되므로 보증금 반환을 받을 수 있다.
-> 보증금 반환 효력이 없어 보증금 반환이나 청구 소송을 진행해야함
## 5. 단독다가구, 연립다세대 연도별 현황
전세사고 건수 많은 3개구(빨강) : 강서구, 구로구, 양천구  
전세사고 건수 적은 3개구(파랑) : 종로구, 용산구, 서초구
#### 단독다가구(단독주택), 소유자 단일
- 단독주택 : 일반적으로 1~2층 주택
- 다중주택 : 3개층 이하인 연면적이 660㎡ 이하인 주택
#### 연립다세대(공동주택), 소유자 다수
- 연립주택 : 4개층 이하인 연면적이 660㎡ 초과인 주택
- 다세대주택 : 4개층 이하인 주택 660㎡ 이하
### 5.1 단독다가구
![다운로드 (3)](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/754142c3-2e17-4c3c-9e78-12fa7cdcfcdf)

연도별로 전반적으로 단독다가구는 상승 추세를 띄고 있다. 전세사기건수 상하위를 비교를 했을때 하위3개는 상위3개보다 높은 가격에 형성이 되있다.  
유독 종로구는 2023년에 크게 떨어졌다.  
사고가 많은 강서구는 단독다가구 보증금 평균이 증가를 했다. 
#### 5.1.1 단독다가구 전세가율
단독다가구데이터는 추정할수 있는 매매가를 구할수 없게 되어있었다. 지번, 본번, 부번이 결측값이 많아 추정이 어려웠다.

### 5.2 연립다세대
![다운로드 (4)](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/9a5dfb1f-9944-4e54-9220-325c3cbd88e5)
연립다세대는 2022년도까지 상승하다가 2023년에 전부 떨어졌다. 

#### 5.2.1 연립다세대 전세가율
![newplot](https://github.com/Kim-jy0819/wanted-pre-onboarding-challenge-data/assets/77106134/b08789dc-4af1-41cb-ab9b-d5fc15ddf6aa)
위에 결과 마찬가지로 강서구 가장 전세가율이 높고 용산구가 가장 낮게 나왔다.

##  결론
- 전세가율이 70%이하인 집이면 경매에 넘어가도 받을 수 있는 확률이 높다. (경매에 유찰 2회 까지 고려)
- 단독다가구 같은 경우 건물의 매매가를 알기가 어렵다.(개인 재산 해당) 이 부분에 대해서는 부동산이나 직접 찾아서 적정 전세인지 판단
- 주택 유형은 거주가 되는 주택으로 되어있을 것(근린생활은 대출x)
- 20, 30대가 상대적으로 집을 많이 구하는 시기인데 부동산 관련해서 잘 알아가는 것이 중요
- 위에 시각화 결과로는 비교적 용산구가 전세가율이 낮아 안전하다. 
- 등기부등본을 봤을때 압류, 가압류의 표기가 있는 것을 확인하고, 계약을 할 때 실소유자와 계약하는 사람 확인하기
- 등기부등본을 봤을때 근저당(경매에 넘어갔을때 우선순위) 확인 -> 근저당 높은 집은 의심
- 임대인의 세무적인 부분도 확인 하면 좋을 것 같음
- 계약을 했을때 대항력, 우선변제권을 갖춰야함(전입신고, 확정일자, 계약 종료후 임차권 등기명령)
- 보증보험을 들었는지 확인하고 되도록 가입

**Comment by ghost** [comment]
은수님! 전세 사기의 위험으로부터 사람들을 보호한다는 취지 참 공감이 갑니다.
저에게도 많은 인사이트를 부탁드립니다 ㅎㅎ

**Comment by dinnercoding** [comment]
@OKEunsu 님! 훌륭하게 시각화 잘 해주셨네요~ 연령대별로 20, 30대에서 전세 사기가 많이 발생하고 특히 강서구에서 전세 사기가 심각한 것으로 보이네요~ 다세대, 연립 주택의 고질적인 문제인 전세가율이 80%에 육박한다는 점도 잘 나타난 것 같습니다.

몇가지 피드백 드립니다~
1. 구별로 세로 막대 그래프를 그리셨는데 구가 많다 보니 빽빽하게 담겨 있어 글자를 회전해서 쓰신 부분들이 있는데 가로 막대 그래프를 그리고 막대간 간격과 막대 너비를 조절해보시면 좀 더 여유롭게 그려보실 수 있을 것 같아요! 또 글자를 가로로 쓸 수 있게 된다는 장점도 활용해보시면 좋을 것 같습니다.
2. 추가적인 정보가 있다면 좀 더 유용할 것으로 보이는데요~ 결국 전세 사기를 사전에 피하기 위한다는 관점에서 세입자가 어떤 정보를 살펴봐야 하는지가 있으면 좋을 것 같습니다.  이를테면 위에서 작성해주신 시각화에서 전세가율이 oo% 이상이면 전세 사기를 당할 확률이 높아진다. 젊은 층일수록 OO 정보를 살펴보면 좋을 것이다. 이런 은수님의 생각들이 시각화와 함께 들어가면 내용이 풍성해지지 않을까 합니다.
